dev-raku/App-Prove6/App-Prove6-0.0.12-r2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit rakudo
+
+DESCRIPTION="Run tests through a TAP harness."
+HOMEPAGE="https://raku.land/cpan:LEONT/App::Prove6"
+SRC_URI="mirror://cpan/authors/id//L/LE/LEONT/Perl6/${P}.tar.gz"
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="primaryuri"
```

Comment:
Why restrict?

Reply:
primaryuri is used in my personal overlay to not make unneccesary connections to gentoo servers.

Without primaryuri, src_fetch contacts gentoo servers to check for availability of source files.

I will try to address the rest as much as possible tomorrow.

Reply:
primaryuri doesn't make sense for something you're proposing for ::gentoo, though. So drop it for the PR?

Reply:
I will soon.